ontology/dcatno_informationModel_en.owl

```diff
@@ -1116,6 +1132,74 @@ modelldcatno:Specialization rdf:type owl:Class ;
                                        "Spesialisering"@nb .
 
 
+#################################################################
+#    Individuals
+#################################################################
+
+###  http://example.com/test#A
+ex:A rdf:type owl:NamedIndividual ,
+              modelldcatno:ObjectType ;
+     modelldcatno:hasProperty ex:ab .
+
+
+###  http://example.com/test#B
+ex:B rdf:type owl:NamedIndividual ,
+              modelldcatno:ObjectType ;
+     modelldcatno:hasProperty ex:ba .
+
+
+###  http://example.com/test#C
+ex:C rdf:type owl:NamedIndividual ,
+              modelldcatno:ObjectType ;
+     modelldcatno:hasProperty ex:cd .
+
+
+###  http://example.com/test#D
+ex:D rdf:type owl:NamedIndividual ,
+              modelldcatno:ObjectType ;
+     modelldcatno:hasProperty ex:dc .
+
+
+###  http://example.com/test#ab
+ex:ab rdf:type owl:NamedIndividual ,
+               modelldcatno:Role ;
+      modelldcatno:formsSymmetryWith ex:ba ;
+      modelldcatno:hasObjectType ex:B ;
+      dct:title "B sin rolle" ;
+      xsd:maxOccurs "*"^^xsd:string ;
+      xsd:minOccurs "1"^^xsd:int ;
+      modelldcatno:symmetryLabel "Navn på relasjonen (Kan settes på ab og/eller ba)"@nb .
```

Comment:
Denne er her bare for å kunne navngi rolleparet. I UML vil dette være navnet på assosiasjonen.